Failed to build twitter_streaming_location (Java)

```
$ cd jubatus-example/twitter_streaming_location/java
$ mvn compile
[INFO] Scanning for projects...
 ... snip ...
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /home/rimms/jubatus-example/twitter_streaming_location/java/src/main/java/example/twitterstreaminglocation/LocationClassifierApp.java:[3,46] error: cannot find symbol
[ERROR]   symbol:   class JubatusClassifierHelper
  location: package example.twitterstreaminglocation
/home/rimms/jubatus-example/twitter_streaming_location/java/src/main/java/example/twitterstreaminglocation/LocationClassifierApp.java:[3,0] error: static import only from classes and interfaces
[ERROR] /home/rimms/jubatus-example/twitter_streaming_location/java/src/main/java/example/twitterstreaminglocation/LocationTrainerApp.java:[3,46] error: cannot find symbol
[ERROR]   symbol:   class JubatusClassifierHelper
  location: package example.twitterstreaminglocation
/home/rimms/jubatus-example/twitter_streaming_location/java/src/main/java/example/twitterstreaminglocation/LocationTrainerApp.java:[3,0] error: static import only from classes and interfaces
[INFO] 4 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
 ... snip ...
```

--- a/twitter_streaming_location/java/src/main/java/example/twitterstreaminglocation/LocationClassifierApp.java
+++ b/twitter_streaming_location/java/src/main/java/example/twitterstreaminglocation/LocationClassifierApp.java
@@ -1,7 +1,5 @@
 package example.twitterstreaminglocation;
 
-import static example.twitterstreaminglocation.JubatusClassifierHelper.list;
-
 import java.io.BufferedReader;
 import java.io.InputStreamReader;
 import java.util.Arrays;

--- a/twitter_streaming_location/java/src/main/java/example/twitterstreaminglocation/LocationTrainerApp.java
+++ b/twitter_streaming_location/java/src/main/java/example/twitterstreaminglocation/LocationTrainerApp.java
@@ -1,7 +1,5 @@
 package example.twitterstreaminglocation;
 
-import static example.twitterstreaminglocation.JubatusClassifierHelper.list;
-
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashMap;

